File: PHP/1-SRP/SRP-Violacao/Cliente.php
<?php
class Cliente
{
    public $id;
    public $nome;
    public $email;
    public $cpf;

    public function adicionarCliente()
    {
        if (!strpos($this->email, '@')) {
            return "Cliente com e-mail inválido";
        }

        if (strlen($this->cpf) != 11) {
            return "Cliente com CPF inválido";
        }

        $PDO = new PDO( 'mysql:host=' . MYSQL_HOST . ';dbname=' . MYSQL_DB_NAME, MYSQL_USER, MYSQL_PASSWORD );
        $sql  = "INSERT INTO CLIENTE (nome, email, cpf) VALUES (:nome, :email, :cpf)";
        $stmt = $DB->prepare($sql);
        $stmt->bindParam(':nome', $nome, PDO::PARAM_STR);
        $stmt->bindParam(':email', $email, PDO::PARAM_STR);
        $stmt->bindParam(':cpf', $cpf, PDO::PARAM_STR);
        $stmt->execute();

        $email = new Email($config);
        $email->destinatario($this->nome, $this->email);
        $email->assunto('Bem Vindo.');
        $email->mensagem('Parabéns! Você está cadastrado.');
        $email->enviar();

        return "Cliente cadastrado com sucesso!";
    }
}

File: Python/1-SRP/SRP-Violacao/Cliente.py
class Cliente:
    def __init__(self, id, nome, email, cpf):
        self.id = id
        self.nome = nome
        self.email = email
        self.cpf = cpf

    def adicionarCliente(self):
        if ('@' not in self.email):
            return 'Cliente com e-mail inválido'

        if (len(self.cpf) != 11): 
            return "Cliente com CPF inválido"

        PDO = PDO('mysql:host=' +  MYSQL_HOST + ';dbname=' + MYSQL_DB_NAME, MYSQL_USER, MYSQL_PASSWORD)
        sql  = "INSERT INTO CLIENTE (nome, email, cpf) VALUES (:nome, :email, :cpf)"

        stmt = DB().prepare(sql)
        stmt.bindParam(':nome', self.nome, PDO.PARAM_STR)
        stmt.bindParam(':email', self.email, PDO.PARAM_STR)
        stmt.bindParam(':cpf', self.cpf, PDO.PARAM_STR)
        stmt.execute()

        Email = Email(config)
        Email.destinatario(self.nome, self.email)
        Email.assunto('Bem Vindo.')
        Email.mensagem('Parabéns! Você está cadastrado.')
        Email.enviar()

        return "Cliente cadastrado com sucesso!"
    
        